fe/fe-core/src/test/java/org/apache/doris/planner/OlapTableSinkTest.java

```diff
@@ -206,7 +206,7 @@ public void testListPartition(
         };
 
         OlapTableSink sink = new OlapTableSink(dstTable, tuple, Lists.newArrayList(p1.getId()), false);
-        sink.init(new TUniqueId(1, 2), 3, 4, 1000, 1, false, false);
+        sink.init(new TUniqueId(1, 2), 3, 4, 1000, 1, false, false, false);
```

Comment:
Is it possiable to add sume UT to cover the logics you added in OlatTableSink?

Reply:
the regression tests have covered them